main.tf

```diff
@@ -1,7 +1,13 @@
 locals {
-  attach_policy = var.attach_elb_log_delivery_policy || var.attach_lb_log_delivery_policy || var.attach_deny_insecure_transport_policy || var.attach_policy
+  attach_policy = var.attach_elb_log_delivery_policy || var.attach_lb_log_delivery_policy || var.attach_cloudtrail_log_delivery_policy || var.attach_deny_insecure_transport_policy || var.attach_policy
+
+  policy_elb_log_prefix        = var.policy_elb_log_prefix != "" ? "${var.policy_elb_log_prefix}/" : ""
```

Comment:
Use this to make sure that there is one slash at the end: "${trimsuffix(var.policy_elb_log_prefix, "/")}/"